application/templates/partials/feedback.html

```
@@ -1,7 +1,12 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-    <h3 class="govuk-heading-s govuk-!-margin-bottom-2 govuk-!-margin-top-0">Help improve this data</h3>
-    <p class="govuk-body"><a href="https://forms.microsoft.com/e/VfSR3Jp8Nu?origin=lprLink&r315037aa6db34b1fa96f65739afca51a={{ dataset["name"] }}" class="govuk-link">Give feedback on this dataset</a>, or email your questions and corrections to <a href="mailto:{{ templateVar.email }}" class="govuk-link">{{ templateVar.email }}</a>.</p>
+    {% if dataset is defined and dataset %}
+      <h3 class="govuk-heading-s govuk-!-margin-bottom-2 govuk-!-margin-top-0">Help improve this data</h3>
+      <p class="govuk-body"><a href="https://forms.microsoft.com/e/VfSR3Jp8Nu?origin=lprLink&r315037aa6db34b1fa96f65739afca51a={{ dataset["name"] }}" class="govuk-link">Give feedback on this dataset</a>, or email your questions and corrections to <a href="mailto:{{ templateVar.email }}" class="govuk-link">{{ templateVar.email }}</a>.</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)